doc/plugins.rst

```diff
@@ -85,8 +85,10 @@ it like:
 .. code-block:: console
 
     $ cd </path/to/tox-fireworks>
-    $ python setup.py sdist register upload
+    $ python setup.py sdist bdist_wheel upload
 
```

Comment:
there's a typo in this commit message, can be left as it is if you do a squash merge, otherwise I suggest a reword 😄

Reply:
Imprve definitely needs improving :)